data-model/src/main/scala/za/co/absa/enceladus/model/Validation.scala

```diff
@@ -13,14 +13,18 @@
  * limitations under the License.
  */
 
-package za.co.absa.enceladus.menas.models
+package za.co.absa.enceladus.model
 
 import com.fasterxml.jackson.annotation.JsonIgnore
 
 object Validation {
 
   val NotSpecified = "not specified"
 
+  val empty = Validation()
+
+  def merge(a: Validation, b: Validation) : Validation = a merge b
```

Comment:
I have a feeling that on one CQC we said we would not use infix notation. Might be wrong. @benedeki ?

Reply:
Fixed, thanks.